engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -520,20 +520,20 @@ class PeriodicProcessService(
 
   }
 
-  private def scheduledExecutionStatus(status: PeriodicProcessDeploymentStatus): ScheduledExecutionStatus = {
+  private def scheduledExecutionStatus(status: PeriodicProcessDeploymentStatus): Option[ScheduledExecutionStatus] = {
     status match {
       case PeriodicProcessDeploymentStatus.Scheduled =>
-        ScheduledExecutionStatus.Scheduled
+        None
```

Comment:
Only periodic deployments in terminal state - with success or error result - are now returned from the periodic deployment manager.